lib/Conversion/FIRRTLToRTL/LowerToRTL.cpp

```diff
@@ -1025,6 +1036,50 @@ LogicalResult FIRRTLLowering::setLoweringTo(Operation *orig,
   return setLowering(orig->getResult(0), result);
 }
 
+/// Create, if needed an always block for the given clock value
+void FIRRTLLowering::inAlwaysForClock(Value clock, std::function<void()> f) {
```

Comment:
I'm not sure if these functions will be necessary after the above comment, but if so, please give them a verb'y name, like "emitAlwaysForClock"

Reply:
Dropping the always merging for now, given complication in previous comment.